### DISCUSSION TITLE
Remove ID from field name when reporting validation errors

### DISCUSSION THREAD

**Discussion by johnngondi** [laravel/framework#51548] [Ideas]
Hello @taylorotwell, I am John, thank you for Laravel Framework, we appreciate the work being done.
Everytime I validate a field where I am selecting something, say am selecting Facility Space, and I submit without selecting an item, the error message reported is like `Facility space id field is required`, is it possible to just say `Facility Space field is required` or just `Facility space is required`

See below

```json
"message": "The facility space id field is required when type is space.",
    "errors": {
        "facility_space_id": [
            "The facility space id field is required when type is space."
        ]
    }
```

can be 

```json
"message": "The facility space is required when type is space.",
    "errors": {
        "facility_space_id": [
            "The facility space is required when type is space."
        ]
    }
```

I know I can do adjustments by publishing the language files. But I feel it can be that way by default.

This is just a thought. No pressure, whatever is there is working good too.

**Comment by ghost** [comment]
Hi. I would not want this change in the core. What if the field is named just id?

**Reply by johnngondi** [reply]
they will remove it like `_id` not just `id` I don't think anyone can call a field  `_id` only.

**Reply by ghost** [reply]
If I were you, I would code this solution for my self, by overwriting stuff from the core validator using macros.
